docs/docs/60-new-docs/90-deprecations.md

```diff
@@ -1,11 +1,52 @@
 ---
 sidebar_label: Deprecations
+description: Overview of deprecated features, their removal timeline, reasons for deprecation, and suggested replacements.
 ---
 
 # Deprecation Notices
 
-:::warning
-Faeka to add notes about deprecations in every minor release >= 1.0 in this
-section. They can we extracted from the release notes and should also link
-to the relevant release notes.
-:::
+As Kargo continues to evolve, features and functionalities are periodically updated,
+improved, or replaced to enhance the platform's capabilities.
+When these changes occur, older features are deprecated and scheduled for removal.
+
+This page provides a list of features that have been deprecated, along with
+their scheduled removal dates and suggested replacements, if any.
+
+## Removed items by Release
+
+### [v1.2.0](https://github.com/akuity/kargo/releases/tag/v1.2.0)
+
+No deprecated features in this release. The focus was on introducing new features like `PromotionTasks`, `Soak Time`, and enhancements to
+promotion steps, along with UI and chart improvements.
```

Comment:
Let's not bother saying anything about releases that _didn't_ either deprecate something or complete the removal of something previously deprecated.